#include "ui_buildconfigurationspage.h"

#if defined(_MSC_VER) && 1000 < _MSC_VER
#pragma once
#endif // defined(_MSC_VER) && 1000 < _MSC_VER

class BuildConfigurationsPage
	: public QWidget
{
	Q_OBJECT

	Ui::BuildConfigurationsPage* ui;

public:

	BuildConfigurationsPage(QWidget *parent = 0);

protected:

public slots:

private:

};
